koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -961,24 +961,23 @@ def handler(self, src, target, opts=None):
         archlist = self.getArchList(build_tag)
 
         flatpak = opts.get('flatpak', False)
+        release_overwrite = opts.get('release')
+
         if flatpak:
             if not osbs_flatpak_support:
                 raise koji.BuildError("osbs-client on koji builder doesn't have Flatpak support")
-            release_overwrite = None
+
+            expected_nvr = None
         else:
             label_overwrites = {}
-            release_overwrite = opts.get('release')
             if release_overwrite:
                 label_overwrites = {LABEL_NAME_MAP['RELEASE'][0]: release_overwrite}
             component, expected_nvr = self.checkLabels(src, label_overwrites=label_overwrites,
                                                        build_tag=build_tag)
 
-        # scratch builds do not get imported, and consequently not tagged
-        if not self.opts.get('scratch') and not flatpak:
-            self.check_whitelist(component, target_info)
-
-        if flatpak:
-            expected_nvr = None
+            # scratch builds do not get imported, and consequently not tagged
+            if not self.opts.get('scratch'):
+                self.check_whitelist(component, target_info)
```

Comment:
Is this indented correctly? Shouldn't it be one level up?

Reply:
we don't have dockerfile => component for flatpak, so check_whitelist is required only for non-flatpak and non-scratch